registry/handlers/manifests.go

```diff
@@ -65,17 +66,58 @@ type manifestHandler struct {
 
 // GetManifest fetches the image manifest from the storage backend, if it exists.
 func (imh *manifestHandler) GetManifest(w http.ResponseWriter, r *http.Request) {
-	ctxu.GetLogger(imh).Debug("GetImageManifest")
+	ctxu.GetLogger(imh).Debug("GetManifest")
```

Comment:
yes.. I thought I combined it and deleted the old one .. must have forgotten to delete the old one